lib/src/user.dart

```diff
@@ -76,6 +81,18 @@ class User {
     return realmCore.userGetProfileData(this);
   }
 
+  /// Gets the refresh token for this [User]. This is the user's credential for
+  /// accessing Atlas App Services and should be treated as sensitive data.
+  String get refreshToken {
+    return realmCore.userGetRefreshToken(this);
+  }
+
+  /// Gets the access token for this [User]. This is the user's credential for
+  /// accessing Atlas App Services and should be treated as sensitive data.
+  String get accessToken {
```

Comment:
It will be good to tests also accessToken in the expected results of some of the tests.

Reply:
Using the access token is a bit of a pain - I can test that it's not empty, but anything more than that will be more complicated than I'd want it to be.

Reply:
I think it is enough, we will be sure that the get method works.